### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by ahmadsarfrazx**
- Added a new branch named desktop-version
- Created index.html
- Created style.css
- No linter error
- Everything is working fine
- Used grid instead of flexbox

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+.jobs {
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  left: 0;
+  top: 718px;
+}
```
Comment (medaminedev66) [inline style.css]: - [ ] Following the project requirements, you should use both Flexbox and Grid in order to lay out the elements on the page. In the previous milestones, you were requested to use "grid" for the "works" section. Kindly use Grid for this milestone as well.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
@ahmadsarfrazx
### This is a manual review
# Request Changes 
*Check the comments below the review.*


Cheers and Happy coding!:clap::clap::clap:

**Comment by ahmadsarfrazx** [comment]
- Used grid instead of flexbox

**Review by medaminedev66 — APPROVED** [review]
@ahmadsarfrazx
### This is a manual review
# Approved :+1: 
Cheers and Happy coding!:clap::clap::clap:

**Pull request merged**